bonobo/settings.py

```diff
@@ -108,7 +108,7 @@ def clear(self):
 
 
 def check():
-    if DEBUG.get() and QUIET.get():
+    if DEBUG.get() and PRINT_NODE_SUMMARY.get():
```

Comment:
this is somewhat awkward since `PRINT_NODE_SUMMARY` and `QUIET` have opposite meanings -- are `DEBUG` and `PRINT_NODE_SUMMARY` in conflict with each other or should we just remove this check?

Reply:
yep, was reading by commit, lgtm